src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -80,6 +93,9 @@ public CustomGamesController(UiService uiService, GameService gameService, Prefe
   }
 
   public void initialize() {
+    gamesTableController = uiService.loadFxml("theme/play/games_table.fxml");
+    gamesTilesContainerController = uiService.loadFxml("theme/play/games_tiles_container.fxml");
```

Comment:
This is the only thing that bugs me about this PR; we're loading both controllers even though we're only displaying one.
The client already uses more memory than I want it to, so if we can found a better solution I'd appreciate that. I didn't take the time yet to take a closer look yet and if I do, I'd like to discuss it directly in Discord ;)